test/test_linop.jl

```diff
@@ -551,6 +551,36 @@ function test_linop()
       @test Matrix([adjtrans(opA)  opA; opA adjtrans(opA)]) == Matrix([adjtrans(A)  A; A adjtrans(A)])
     end
   end
+
+  @testset ExtendedTestSet "Counters" begin
+    op = LinearOperator(rand(3,4) + im * rand(3,4))
+    @test nprod(op) == 0
+    @test ntprod(op) == 0
+    @test nctprod(op) == 0
+    nprods = 5
+    ntprods = 4
+    nctprods = 7
+    for _ = 1 : nprods
+      op * rand(4)
+    end
+    for _ = 1 : ntprods
+      transpose(op) * rand(3)
+    end
+    for _ = 1 : nctprods
+      op' * rand(3)
+    end
+    @test nprod(op) == nprods
+    @test ntprod(op) == ntprods
+    @test nctprod(op) == nctprods
```

Comment:
Could add a test here for transpose, adjoint and conjugate operators? So we can cover [these lines](https://coveralls.io/builds/26961540/source?filename=src/adjtrans.jl#L31). :100:

Reply:
Good idea. Done. Thanks.